types/bree/index.d.ts

```diff
@@ -0,0 +1,68 @@
+// Type definitions for bree 6.2
+// Project: https://github.com/breejs/bree
+// Definitions by: Taylor Schley <https://github.com/shadogate15>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/*~ This declaration specifies that the class constructor function
+ *~ is the exported object from the file
+ */
+
```

Comment:
```suggestion
```
these are instruction from the template for the writer

Reply:
removed